library_controller/statistics.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+# rubocop:disable LineLength
```

Comment:
don't do that thing.
You need to read a documentation of `Rubocop`.
You must create `rubocop.yml` file and there put your configs